### PR TITLE
docs: Add server urls to swagger-ui

### PR DESCRIPTION
**Pull request by PierreBesson**
Add public sidecar URLs to swagger-ui to let people try out API requests. CORS headers have been set up on the server-side to allow requests from https://paritytech.github.io/ to be accepted.

### CODE REVIEW

File: docs/src/index.js [hunk 1]
```diff
@@ -6,6 +6,5 @@ const spec = require('./openapi-v1.yaml');
 const ui = SwaggerUI({
   spec,
   dom_id: '#spec',
-  supportedSubmitMethods: []
+  supportedSubmitMethods: ["get", "post"]
```
Comment (Imod7) [inline docs/src/index.js:9]: Thank you so much @PierreBesson for the contribution and adding the `servers` option! 💯
However, currently the `Try it out` feature in swagger UI of Sidecar is disabled due to a CORS issue : 
- related sidecar [comment](https://github.com/paritytech/substrate-api-sidecar/issues/1190#issuecomment-1398856686) and related [PR](https://github.com/paritytech/substrate-api-sidecar/commit/22f8fd47ee2ea0cc0662d6ad8ff36fe9db68674d)
- error received when using the `Try it out` button (if enabled e.g. while connected in polkadot public sidecar) 
  ```
  Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://polkadot-public-sidecar.parity-chains.parity.io/runtime/code. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 200.
  ```
- related [resource](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin?utm_source=devtools&utm_medium=firefox-cors-errors&utm_campaign=default) : _"Reason: CORS header 'Access-Control-Allow-Origin' missing"_

So I am not sure if at this moment we would like to enable it again and tackle this CORS error.

cc @TarikGul

Comment (TarikGul) [reply docs/src/index.js:9]: Via Pierres comment above CORS headers have been added on the Server-side for `https://paritytech.github.io/` so we should be totally fine!

**Comment by TarikGul** [comment]
> CORS headers have been set up on the server-side to allow requests from https://paritytech.github.io/ to be accepted.

Thanks so much for setting this up!

**Review by TarikGul — APPROVED** [review]
Just the merge conflict needs to be resovled, but thanks so much again! LGTM

**Review by Imod7 — APPROVED** [review]

File: docs/src/index.js (hunk 1 above)
Comment (Imod7) [reply docs/src/index.js:9]: My bad! I did not read the description!

Comment (PierreBesson) [reply docs/src/index.js:9]: Yes CORS has been dealt with.

**Pull request merged**